### PR TITLE
Fix DelegateProxy to emit values to multiple subscribers

### LINKED ISSUE #46: `didSelectItemPublisher` from collection view only pushes element to latest subscriber
**Describe the bug**
I had 2 subscribers subscribed to `didSelectItemPublisher` of collection view. One to open another controller, one to call api. But when I tapped on collection view cell, it just emited elements to the latest subscriber, ignored the previous one.

**To Reproduce**
Subscribe 2 subscribers to `didSelectItemPublisher` of collection view and see what happens

**Expected behavior**
`didSelectItemPublisher` emits to all its subscribers

**Device:**
 - Device: iPhone 11 pro
 - OS: macOS BigSur
 - CombineCocoa version: latest

### PR DESCRIPTION
**Pull request by TTOzzi**
Fixes #46

### CODE REVIEW

File: Example/ExampleTests/UICollectionViewTests.swift [hunk 1]
```diff
@@ -70,18 +74,38 @@ class UICollectionViewTests: XCTestCase {
...
+
+        var resultIndexPaths = [IndexPath]()
+
+        collectionView.didSelectItemPublisher
+            .sink(receiveValue: { resultIndexPaths.append($0) })
+            .store(in: &subscriptions)
+
+        collectionView.didSelectItemPublisher
+            .sink(receiveValue: { resultIndexPaths.append($0) })
+            .store(in: &subscriptions)
```
Comment (freak4pc) [inline Example/ExampleTests/UICollectionViewTests.swift:103]: Preferably store to two different arrays and make sure they're equal, would be a more realistic test

File: Example/ExampleTests/UITableViewTests.swift [hunk 2]
```diff
@@ -86,12 +94,33 @@ class UITableViewTests: XCTestCase {
...
         let givenIndexPath = IndexPath(row: 1, section: 0)
         tableView.delegate!.tableView!(tableView, accessoryButtonTappedForRowWith: givenIndexPath)
 
         XCTAssertEqual(resultIndexPath, givenIndexPath)
     }
+    
+    func test_didSelectRowAt_for_multiple_subscribers() {
+        let tableView = UITableView()
+
+        var resultIndexPaths = [IndexPath]()
```
Comment (freak4pc) [inline Example/ExampleTests/UITableViewTests.swift:110]: same about making two vars here

**Review by freak4pc — APPROVED** [review]
Looks great! A few notes on the tests but asides for that I'm happy to merge it.
Thanks for taking care of it :)

**Comment by TTOzzi** [comment]
Thanks for the feedback @freak4pc. Committed your suggested changes 😁

**Pull request merged**